shared_python/Tags.py

```diff
@@ -30,12 +31,15 @@ def __init__(self, args, db, log):
 
 
   def create_tags_table(self, database = None):
+    """
+    Used only in step 02 for non-eFiction archives
+    """
     try:
       database = self.database if database is None else database
-      self.cursor.execute("DROP TABLE IF EXISTS {0}.`tags`".format(database))
+      self.sql.execute("DROP TABLE IF EXISTS {0}.`tags`".format(database))
```

Comment:
Does this work? `sql` here would be the SQL object itself; I think it should be `self.sql.cursor.execute()`.

Reply:
JOKES I can't read this is fine

Reply:
I must admit that calling the custom method `execute` was not my smartest move! Maybe we can do some more refactoring once this codebase is working all the way through